about.css

```diff
@@ -0,0 +1,539 @@
+/* about page */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "lato", sans-serif;
+  min-height: 100vh;
+  background-color: #e5e5e527;
+}
+
+.social-media,
+.navbar {
+  display: none;
+}
+
+.about-box {
+  background: url("./images/home-page.png") no-repeat;
+  background-size: cover;
+  margin-bottom: 50px;
+}
+
+.hamburger {
+  border: none;
+  background: transparent;
+  position: relative;
+  margin-top: 8%;
+  padding-left: 5%;
+}
+
+.hamburger img {
+  width: 60%;
+  height: 70%;
+  margin-bottom: 80%;
+}
+
+.about-intro {
+  position: relative;
+  margin-left: 5%;
+  font-size: 110%;
+  margin-right: 15%;
+  font-weight: normal;
+}
+
+.hello {
+  position: relative;
+  color: #f35826;
+  font-size: 200%;
+  margin-bottom: 0%;
+  padding-left: 5%;
+}
+
+.heading {
+  position: relative;
+  color: #f35826;
+  font-size: 400%;
+  margin-top: 0%;
+  font-weight: bold;
+  padding-left: 5%;
+}
+
+.mission {
+  position: relative;
+  border: #f2f2f2 2px solid;
+  background-color: #fff;
+  text-align: center;
+  padding: 4%;
+  margin-right: 3%;
+  margin-left: 3%;
+  font-size: 110%;
+  opacity: 0.7;
+}
+
+.contact {
+  position: relative;
+  text-align: center;
+  margin: 10%;
+}
+
+.email {
+  text-decoration: underline;
+  font-weight: bold;
+  opacity: 0.8;
+}
+
+.logo-heading {
+  position: relative;
+  text-align: center;
+  padding-top: 5%;
+  padding-bottom: 2%;
+}
+
+.logo-heading::after {
+  content: '';
+  height: 2px;
+  width: 5%;
+  background: #f35826;
+  position: absolute;
+  bottom: 0;
+  left: 48%;
+}
+
+.logo-creation {
+  text-align: center;
+  margin-top: 20px;
+}
+
+.logo-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: #f2f2f2 2px solid;
+  padding: 5%;
+  margin-right: 3%;
+  margin-left: 3%;
+  margin-top: 5%;
+}
+
+.img-logo {
+  width: 200px;
+}
+
+.summit {
+  font-weight: bold;
+  margin-bottom: 0%;
+}
+
+.logo-text {
+  margin-left: 2%;
+}
+
+.period {
+  border-top: grey solid 1px;
+  border-bottom: grey solid 1px;
+  margin-top: 0%;
+  font-size: 75%;
+}
+
+/* past summits section */
+.previous-summits {
+  border-top: #f2f2f2 2px solid;
+  margin-top: 5%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.previous-heading,
+.previous-text {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.previous-div {
+  height: 2px;
+  width: 10%;
+  background: #f35826;
+}
+
+.previous-text {
+  text-align: center;
+}
+
+.previous-list {
+  list-style-type: none;
+}
+
+.previous-list li {
+  border: none;
+  margin-bottom: 7%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 80px;
+  color: #fff;
+  text-align: center;
+}
```

Comment:
I'm experiencing a horizontal scroll in the mobile version of the `About` page that makes for bad UI/UX. Please fix it.